packages/frontend/src/pages/Projects.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { FC } from 'react';
```

Comment:
WIP page to provide project selection if the user logs in to root and has multiple projects available. 

Nobody is going to see this page until we allow multiple projects to be created, it redirects if only 1 project is available.